### PR TITLE
[HOTFIX - MERGE WITH GITFLOW] Audit report redirects [Estimated deploy date: 2/18/2021)

### PR DESCRIPTION
**Pull request by patphongs**
Adds transition redirects for:

- Individual Audit Report page redirects
- Audit report landing page redirects
- Audit report PDF file redirects

This hotfix is estimated to be deployed on 2/18/2021

### CODE REVIEW

File: redirects-transition.conf [hunk 1]
```diff
@@ -942,6 +1074,14 @@ rewrite ^/agenda/agendas2000/(.*).pdf https://www.fec.gov/resources/updates/agen
 # audio/ broader redirects
 rewrite ^/audio/([0-9]+)/(.*) https://www.fec.gov/resources/audio/$1/$2 redirect;
 
+# audits/ broader redirects for PDF files
+rewrite ^/audits/Hillary_Clinton_for_President/(.*) https://www.fec.gov/resources/legal-resources/enforcement/audits/Hillary_Clinton_for_President/$1;
```
Comment (dorothyyeager) [inline redirects-transition.conf:1078]: It looks like all of the broader redirects for audit PDF files are missing the "redirect;" part at the end.

**Review by dorothyyeager — CHANGES_REQUESTED** [review]
See comment about the broader PDF redirects...

File: redirects-transition.conf (hunk 1 above)
Comment (patphongs) [reply redirects-transition.conf]: Nice find @dorothyyeager! This has been fixed

**Review by dorothyyeager — APPROVED** [review]
Looks good @patphongs!

**Pull request merged**